components/notebook-controller/controllers/notebook_controller.go

```diff
@@ -345,22 +461,8 @@ func generateStatefulSet(instance *v1beta1.Notebook) *appsv1.StatefulSet {
 			},
 		}
 	}
-	container.Env = append(container.Env, corev1.EnvVar{
-		Name:  "NB_PREFIX",
-		Value: "/notebook/" + instance.Namespace + "/" + instance.Name,
-	})
 
-	// Add readiness probe
-	if container.ReadinessProbe == nil {
-		container.ReadinessProbe = &corev1.Probe{
-			Handler: corev1.Handler{
-				HTTPGet: &corev1.HTTPGetAction{
-					Path: "/notebook/" + instance.Namespace + "/" + instance.Name,
-					Port: intstr.FromInt(DefaultContainerPort),
-				},
-			},
-		}
-	}
```

Comment:
We should keep this https://github.com/StatCan/kubeflow/pull/85

Reply:
Now I'm getting an error, 
![image](https://user-images.githubusercontent.com/8212170/190710368-64194f23-ddba-44b4-bd0e-e43d419b9030.png)

Reply:
must have been renamed or changed then, ill look

Reply:
[ProbeHandler ](https://github.com/kubernetes/api/blob/c98ebf18d36620d52158313ac5c992e07955ac7f/core/v1/types.go#L2193)
![image](https://user-images.githubusercontent.com/23174198/190713213-0cad4ccb-358b-40d5-aedf-10cf65181f66.png)

Reply:
oh it will stay as the same `corev1` i believe it would look like this instead
![image](https://user-images.githubusercontent.com/23174198/190801083-83bb04dc-29af-431f-b824-cd6ef236b250.png)